src/main/java/com/thevoxelbox/voxelsniper/voxelsniper/material/BukkitMaterial.java

```diff
@@ -36,6 +36,11 @@ public String getName() {
         return material.name();
     }
 
+    @Override
+    public boolean equals(VoxelMaterial material) {
```

Comment:
Not entirely sure on the equals, I haven't seen anyone make an overloaded equals before, I think most people Override equals(Object) instead?